### DISCUSSION TITLE
How to support Screen Split on iOS?

### DISCUSSION THREAD

**Discussion by GeneralVimes** [airsdk/Adobe-Runtime-Support#3502] [Q&A] [answered]
I found that my AIR games, if run on Android in the split screen more, can take their screen section perfectly, without any additional work from me.

But when I try to open my game on iPadOS in the split screen mode, it takes the whole screen and a message is shown "Split Screen is not supported". Can anything be done about it? Perhaps some addition to application.xml?

**Comment by GeneralVimes** [comment]
Here is how it works on Android: https://imgur.com/VRx3itC

**Comment by jamikado** [comment, marked as answer]
Just searching the issues here shows https://github.com/airsdk/Adobe-Runtime-Support/issues/321#issuecomment-627341705 where someone got it working, maybe first try the suggestions from there.

**Reply by GeneralVimes** [reply]
Thank you! It worked like a charm!
Now my games can support split-screen on iPadOS, too: https://imgur.com/nc68eT3
